filebeat/docs/getting-started.asciidoc

```diff
@@ -20,10 +21,120 @@ After installing the Elastic Stack, read the following topics to learn how to in
 * <<filebeat-command-line>>
 * <<directory-layout>>
 
+[[filebeat-modules-quickstart]]
+=== Quick Start for Common Log Formats
+
+beta[]
+
+Filebeat provides a set of pre-built modules that you can use to rapidly
+implement and deploy a log monitoring solution, complete with sample dashboards
+and data visualizations, in about 5 minutes. These modules support common log
+formats, such as Nginx, Apache2, and MySQL, and can be run by issuing a simple
+command.  
```

Comment:
It might make sense to add a paragraph somewhere here to explain that people should follow this Quick Start if they want to monitor log files from one of the supported modules. Otherwise, they can skip to "Installing Filebeat".